### PR TITLE
Rename project to LavinMQ

### PR DESCRIPTION
**Pull request by snichme**
TODO: 

- [ ] lavinqmq.com to github pages
- [ ] lavinmq.com emails (support@lavinmq.com)
- [x] packagecloud new repo
- [x] Logo with white text 

image

### CODE REVIEW

**Comment by jage** [comment]
> How to handle Changelog? Rename older version or consider that "append only" log :)

How do you mean, mention of the string "avalanchemq"?

**Comment by snichme** [comment]
> > How to handle Changelog? Rename older version or consider that "append only" log :)
> 
> How do you mean, mention of the string "avalanchemq"?

yes, the change was made before the rename, so was the change made on avalanchemq or lavinmq? ;)

**Comment by jage** [comment]
> yes, the change was made before the rename, so was the change made on avalanchemq or lavinmq? ;)

Thinking we can reset the changelog (i.e. prerelease-changelog) when we release 1.0.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -62,7 +62,7 @@ jobs:
         run: crystal spec --no-color --order random
 
   compile:
-    name: Compile AvalancheMQ
+    name: Compile Lavinmq
```
Comment (dentarg) [inline .github/workflows/ci.yml:65]:
```suggestion
    name: Compile LavinMQ
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -106,10 +106,10 @@ jobs:
           name: bin
           path: bin
 
-      - name: Run avalanchemq in background
+      - name: Run lavinmq in background
```
Comment (dentarg) [inline .github/workflows/ci.yml:109]:
```suggestion
      - name: Run LavinMQ in background
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -140,23 +140,23 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y libevent-dev
 
-      - name: Checkout AvalancheMQ for bunny_definitions.json
+      - name: Checkout Lavinmq for bunny_definitions.json
```
Comment (dentarg) [inline .github/workflows/ci.yml:143]:
```suggestion
      - name: Checkout LavinMQ for bunny_definitions.json
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -140,23 +140,23 @@ jobs:
...
         uses: actions/checkout@v2
 
       - uses: actions/download-artifact@v2
         with:
           name: bin
           path: bin
 
-      - name: Run avalanchemq in background
+      - name: Run lavinmq in background
```
Comment (dentarg) [inline .github/workflows/ci.yml:151]:
```suggestion
      - name: Run LavinMQ in background
```

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -192,10 +192,10 @@ jobs:
...
```
Comment (dentarg) [inline .github/workflows/ci.yml:195]:
```suggestion
      - name: Run LavinMQ in background
```

File: .github/workflows/ci.yml [hunk 6]
```diff
@@ -228,10 +228,10 @@ jobs:
...
```
Comment (dentarg) [inline .github/workflows/ci.yml:231]:
```suggestion
      - name: Run LavinMQ in background
```

File: .github/workflows/ci.yml [hunk 7]
```diff
@@ -265,10 +265,10 @@ jobs:
...
```
Comment (dentarg) [inline .github/workflows/ci.yml:268]:
```suggestion
      - name: Run LavinMQ in background
```

**Review by dentarg — COMMENTED** [review]
Taking the chance to correct some inconsistency that have bugged me :)

**Pull request merged**